accname/name/comp_text_node.html

```diff
@@ -13,23 +13,96 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_text_node">#comp_text_node</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<!-- I'm not certain whether #comp_text_node requires a lot of testing outside of the #comp_name_from_content contexts, -->
-<!-- but I did think of one example where text node versus comment node may make a difference when joining text nodes with a space vs innerText. -->
+<!-- Note: some overlap with the tests in ./comp_name_from_content.html -->
 
-<!-- Skipped (class="ex" removed) until https://github.com/w3c/accname/issues/193 is resolved -->
-<h2 class="ex-skipped" data-expectedlabel="heading label" data-testname="heading with text/comment/text nodes, no space">
-  heading<!-- with non-text node splitting concatenated text nodes -->label<!-- [sic] no extra spaces around first comment -->
-</h2>
 
+<h1>text/element/text nodes, no space</h1>
```

Comment:
Minor suggestion on usage of multiple <h1>s (although unsure if that was intentional).

Reply:
Also curious if there are children text nodes that  are both empty and non-empty, e.g.,:

```
<button>
     button
     <span>label</span>
     <span></span>
     <span>foo</span>
</button>
```

(`computedLabel = buttonlabelfoo`)

Reply:
> Minor suggestion on usage of multiple `h1`s (although unsure if that was intentional).

Is the suggestion to not use multiple H1s? If so, I don't think it matters, but I can change it if you feel strongly about it.

Reply:
> Is the suggestion to not use multiple H1s? If so, I don't think it matters, but I can change it if you feel strongly about it.

Apologies, I didn't mean to be pedantic and definitely don't feel strongly about it in this context (have definitely had one too many spirited discussions about multiple `h1`s 😛?)